app/src/main/java/org/fossasia/openevent/general/about/AboutEventFragment.kt

```diff
@@ -12,24 +12,24 @@ import androidx.lifecycle.Observer
 import androidx.navigation.fragment.navArgs
 import com.google.android.material.appbar.AppBarLayout
 import com.google.android.material.snackbar.Snackbar
+import com.squareup.picasso.Picasso
 import kotlinx.android.synthetic.main.fragment_about_event.view.appBar
 import kotlinx.android.synthetic.main.fragment_about_event.view.progressBarAbout
 import kotlinx.android.synthetic.main.fragment_about_event.view.aboutEventContent
 import kotlinx.android.synthetic.main.fragment_about_event.view.aboutEventDetails
 import kotlinx.android.synthetic.main.fragment_about_event.view.eventName
 import kotlinx.android.synthetic.main.fragment_about_event.view.detailsHeader
-import kotlinx.android.synthetic.main.fragment_about_event.view.aboutEventCollapsingLayout
+import kotlinx.android.synthetic.main.fragment_about_event.view.aboutEventImage
 import org.fossasia.openevent.general.R
 import org.fossasia.openevent.general.event.Event
 import org.fossasia.openevent.general.event.EventUtils
 import org.fossasia.openevent.general.utils.extensions.nonNull
 import org.fossasia.openevent.general.utils.stripHtml
 import org.koin.androidx.viewmodel.ext.android.viewModel
 
-class AboutEventFragment : Fragment(), AppBarLayout.OnOffsetChangedListener {
+class AboutEventFragment : Fragment() {
     private lateinit var rootView: View
     private val aboutEventViewModel by viewModel<AboutEventViewModel>()
-    private var isHideToolbarView: Boolean = false
```

Comment:
Consider renaming this to `hideToolbarView` or `showToolbarView`. `isHideToolbarView` does not seem appropriate in my opinion

Reply:
This red line means I deleted this because it is not necessary

Reply:
My bad sorry about that